src/Pages/ExplorePage/ExplorePage.css

```diff
@@ -0,0 +1,29 @@
+.chips-wrapper {
+  background-color: var(--light-black);
+  border-top: 1px solid var(--body-border);
+  border-bottom: 1px solid var(--body-border);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+.chip-box {
+  background-color: var(--chip-color);
+  color: var(--white);
+  max-width: 6rem;
+  min-width: 5rem;
+  padding: 7px;
+  border-radius: 1rem;
+  border: 1px solid var(--chip-border);
+  margin: 10px;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+}
+/* .chip-box:hover {
+  background-color: var(--chip-hover);
+} */
```

Comment:
remove commented code while pushing to github

Reply:
sure will remove it